Core.Collectors.Tests/Collector/CachingCollectorTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Microsoft.CloudMine.Core.Collectors.Collector.Tests
+{
+    [TestClass]
+    public class CachingCollectorTests
+    {
+        [TestMethod]
+        public void ShallIgnoreCache()
+        {
+            bool ignoreCache = true;
+            bool scheduled = true;
+            DateTime utcNow = DateTime.Parse("2022-05-31T05:30:00");
```

Comment:
This date is not utcNow. Confused. Also not specifying time zones might make the test not run locally in different time zones.